packages/devextreme/js/ui/tabs.d.ts

```diff
@@ -102,6 +103,20 @@ export type OptionChangedEvent<TItem extends ItemLike = any, TKey = any> = Event
  */
 export type SelectionChangedEvent<TItem extends ItemLike = any, TKey = any> = EventInfo<dxTabs<TItem, TKey>> & SelectionChangedInfo<TItem>;
 
+/**
+ * @docid _ui_tabs_SelectionChangingEvent
+ * @public
+ * @type object
+ * @inherits EventInfo,SelectionChangingInfo
+ */
+export type SelectionChangingEvent<TItem extends ItemLike = any, TKey = any> = EventInfo<dxTabs<TItem, TKey>> & SelectionChangingInfo<TItem> & {
+    /**
+     * @docid _ui_tabs_SelectionChangingEvent.cancel
+     * @type Boolean|Promise<Boolean>
+     */
+    cancel: boolean | PromiseLike<boolean>;
```

Comment:
Isn't it duplicated because u already declared cancel in [collection's SelectionChangingInfo](https://github.com/DevExpress/DevExtreme/pull/27954/files#diff-2ad84393a8ade2049c8f6dcaef5a60c99a0e84f57a0314cb0b1db85a3316af8cR28)?

Reply:
https://github.com/DevExpress/DevExtreme/pull/27954/commits/ca78c9791306c8ad783754bf6230ebb18e85db4b